### ISSUE TITLE
Switch repository to use RDF/ttl rather than manchester syntax

### ISSUE THREAD

**Issue by cmungall** [geneontology/noctua-models#14]
Manchester is problematic

 - v sensitive to issues such as all APs needing to be declared - see #12 
 - not as readable or manually editable as advertised
 - hard to parse outside JVM

Candidates

 * OFN
 * RDF/TTL (widely parseable BUT annoying repetition with axiom annotations)

**Comment by kltm** [comment]
This will have to be coordinated with a redo of large parts of epione.js (geneontology/noctua) and likely some bits in minerva (export?) 
@hdietze

**Comment by cmungall** [comment]
Should be a trivial change in Minerva, but yes, will need coordinated.

I'm nominating ttl as the format.

Of all the OWL formats, OFN was really the only contender, due to the pleasingness of generally one-line-per axiom. But ultimately OWL-only formats are too obscure and there will probably never be *reliable* parsers for them that are not the OWLAPI.

RDF formats can be operated on directly via a wide variety of tools. It can be SPARQL-queried directly from the command line, funneled into a triplestore. there are APIs for all languages.

Of the RDF formats, RDF/XML is right out. There are some nice features in the 4-ary variants (e.g. N-Quads) but it's better to stick to the standard 3-ary subset. Of these, ttl is the most widely adopted.

as far as

> a redo of large parts of epione.js

Using N3.js should allow you to switch from whatever regexp hacking you're doing to a clean fast library.

**Comment by kltm** [comment]
Yes, I looked at N3.js and others for this job a while back, which led to the another format switch discussion.

**Comment by cmungall** [comment]
Won't fix. See https://docs.google.com/document/d/1gzdOjmQn34-A2EUJuujoMzVrAZr0RZfofCVjxW3msZU/edit#
